### PR TITLE
Refactor/animal parameters

### PR DESCRIPTION
**Pull request by lorenzosenarigo**

### CODE REVIEW

**Review by Johnny-MB — CHANGES_REQUESTED** [review]
add comment to rates

File: GeoRasterBlueprint/Model/Bison.cs [hunk 1]
```diff
@@ -20,25 +20,43 @@ public class Bison : AbstractAnimal {
     public override double Longitude { get; set; }
 
     #endregion
+    
     #region Constants
     private readonly Dictionary<AnimalLifePeriod, double> _satietyIntakeHourly = new()
```
Comment (Johnny-MB) [inline GeoRasterBlueprint/Model/Bison.cs:25]: rename to starvationRate

File: GeoRasterBlueprint/Model/Bison.cs [hunk 2]
```diff
@@ -20,25 +20,43 @@ public class Bison : AbstractAnimal {
...
     {
-        //food per day (kg) / 16 (hours)
-        { AnimalLifePeriod.Calf, 0.56 }, //9kg per day
-        { AnimalLifePeriod.Adolescent, 1.81 }, //20-29 kg per day
-        { AnimalLifePeriod.Adult, 3.75 } //60 kg per day, 113 liter
+        //value adjusted to maxSatiety = 100
+        { AnimalLifePeriod.Calf, MaxSatiety * DailyFoodAdult / 16 / DailyFoodAdult },
```
Comment (Johnny-MB) [inline GeoRasterBlueprint/Model/Bison.cs:28]: add comment explaining calculation

**Comment by lorenzosenarigo** [comment]
- rename satietyintakehourly
- Edit Calf starvation rate

**Pull request closed without merging**